internal/test/wmcb/wmcb_test.go

```diff
@@ -182,6 +236,151 @@ func (vm *wmcbVM) remoteDownloadExtract(url, sha, remoteDownloadFile, remoteExtr
 	return nil
 }
 
+// initializeTestConfigureCNIFiles initializes the files required for configure-cni
+func (vm *wmcbVM) initializeTestConfigureCNIFiles(ovnHostSubnet string) error {
+	// Create the CNi directory C:\Windows\Temp\cni on the Windows VM
+	_, stderr, err := vm.Run("if not exist "+winCNIDir+" mkdir "+winCNIDir, false)
```

Comment:
Is it worth thinking about the case where `winCNIDir` is a file, not a directory? We can probably assume its not if this is a fresh machine.

Reply:
If this was in the product, then yes I would agree.

Reply:
Actually the way the test works is that if there is a trailing backslash it will check for a directory. If it is not present it will check for a director or file.